scripts/cypress.js

```diff
@@ -0,0 +1,36 @@
+const rl = require('readline')
+const { exec } = require('child_process')
+
+const DEPLOYMENTS = {
+  d: 'https://safe-team.dev.gnosisdev.com/app',
+  s: 'https://safe-team.staging.gnosisdev.com/app',
+  p: 'https://gnosis-safe.io/app',
+}
+
+const readline = rl.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+})
+
+const cmd = `cypress ${process.argv[2]}`
+
+function launchCypressEnv() {
+  readline.question(
+    'Which environment do you want to test: (l)ocal, (d)evelopment, (s)taging or (p)roduction? ',
```

Comment:
Can you also accept it as a CLI argument? So that we can call it like so:
```
yarn cypress:run -d
```

Reply:
We could set the environment before the command and have varying env files with `CYPRESS_BASE_URL`s in them. What do you think?

Reply:
We will have to have extra env files which may overwrite each other. I will look deeper into this tomorrow.

Reply:
Not sure why the discourse shifted to env files. Just accepting an optional [argument](https://www.baeldung.com/linux/use-command-line-arguments-in-bash-script) should be enough.